src/app/core/connection.js

```diff
@@ -482,8 +543,10 @@ export default class Connection {
 
     /**
      * Allows the user to add a sink to the connection
-     * @param component
-     * @param port
+     * @param {Object} component 
```

Comment:
It seems like this needs to be a string. Verifying this needs to be an github issue